api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
I meant to respond to this last night, but didn't have a chance. Sorry for the delay. Having a contributors sub-route or not is fine by me; we do it in the nodes endpoint, but if it's not necessary, that's fine. The important thing is that serialization of any user information has to go through either the users serializer or it has to be otherwise anonymizable. Be sure there's a test for "contributor information does not show if anonymous view only link" or similar. There's a wrapper that will handle the work on a field-by-field basis; you'll see it I in the user serializer.

In general, I was hoping to force all serialization of user information to go through the User serializer, so the longer we can keep that, the better. I'm not going to be super-strict about it as long as that's taken care of.